src/main/java/net/kyori/mu/math/MuMath.java

```diff
@@ -320,4 +321,32 @@ public static long sum(final long... values) {
     }
     return sum;
   }
+
+  /**
+   * Sums up {@code values}.
+   * @param values the values
+   * @return the sum
+   */
+  public static int sum(final List<Integer> values) {
+    return values.stream().mapToInt(i -> i).sum();
+  }
+
+  /**
+   * Gets the maximum value from a List of values.
+   * @param values the values
+   * @return the maximum if the list is non empty, 0 otherwise.
+   */
+  public static int max(final List<Integer> values) {
+    return values.stream().reduce((v1,v2)-> v1 > v2 ? v1 : v2).orElse(0);
```

Comment:
```suggestion
    return values.stream().reduce(Math::max).orElseThrow(NoSuchElementException::new);
```
also need to consider whether to use parallel stream